web/src/main/java/com/marklogic/hub/web/service/EntityManagerService.java

```diff
@@ -134,9 +134,7 @@ public EntityModel saveEntity(EntityModel entity) throws IOException {
         else {
             HubEntity renamedEntity = em.saveEntity(hubEntity, true);
             entity.setFilename(renamedEntity.getFilename());
-
-            // Redeploy the flows
-            dataHubService.reinstallUserModules(hubConfig, null, null);
+            dataHubService.installUserModules(hubConfig, true, null, null);
```

Comment:
There is a FileWatcherService which looks into changes in entities, mappings and flows dir and does installUserModules (with 'forceLoad' set to false). Do we even have to call installUserModules() explicitly here and also when saving mapping and flows ?